Source/Dafny/Util.cs

```diff
@@ -445,4 +445,18 @@ public class DependencyMap
       }
     }
   }
+  
+  class ListComparer<T> : IEqualityComparer<List<T>> {
```

Comment:
Can you make this work for `IEnumerable<T>` instead of `List<T>` ?

Reply:
Done.